vividus-tests/src/main/resources/properties/environment/integration/envinroment.properties

```
@@ -24,3 +24,5 @@ http.service-unavailable-retry.max-retries=2
 
 db.connection.csv-data.driver-class-name=org.relique.jdbc.csv.CsvDriver
 db.connection.csv-data.url=jdbc:relique:csv:classpath:data?separator=;&quoteStyle=C
+
+system.ENV_VAR=ENC(pMLhry0P0zuhIiy/lXAiiqytAAAFJSFNqAY83S9yGf8=)
```

system properties are not the same as environment variables:
```suggestion
system.system-property=ENC(pMLhry0P0zuhIiy/lXAiiqytAAAFJSFNqAY83S9yGf8=)
```